script.js

```diff
@@ -2,3 +2,83 @@ var main = function (input) {
   var myOutputValue = 'hello world';
   return myOutputValue;
 };
+// set global var//
+var win = 0;
+var lose = 0;
+var draw = 0;
+var troll = 0;
+
+// hack function//
```

Comment:
This is a useless comment. It doesn't explain anything, only repeat the function name. Comments should explain why code exists and what it does in English.